Resources/Prototypes/_Floof/Recipes/Lathes/clothing.yml

```
@@ -77,3 +77,14 @@
   materials:
     Cloth: 250 # Two utility belts plus some connecting straps
     Steel: 50 # Because buckles
+
+- type: latheRecipe
+  id: ClothingUniformNudityPermit
+  icon:
+    sprite: _Floof/Clothing/Uniforms/nuditypermit.rsi
+    state: icon
+  result: ClothingUniformNudityPermit
+  category: ClothingOther # Floof - add uniform printer clothing categories
+  completetime: 2
+  materials:
+    Durathread: 100
```

Custom icon shouldn't be necessary since it will be inherited from the resulting entity, and floof comment is not needed either because it's in the floof namespace
```suggestion
- type: latheRecipe
  id: ClothingUniformNudityPermit
  result: ClothingUniformNudityPermit
  category: ClothingOther
  completetime: 2
  materials:
    Durathread: 100
```